README.md

```diff
@@ -26,33 +56,173 @@ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+* AWS CLI version 2+ [_See_ AWS CLI Installation Guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html)
+* AWS Profile Access \
+-- <u>Note</u>: Engineers *must* have access and permissions to create AWS resources
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.2 Clone DIBBS-AWS Repository
+
+<em><strong>4.2.1.</em></strong> Create a directory to store the repository on your local machine
+- Mac OS users: `mkdir workspace`
+- Windows users: `md workspace`
+
+<em><strong>4.2.2.</em></strong> Clone the dibbs-aws repository (<u>use one of the following commands</u>:)\
+&nbsp;&nbsp;&nbsp;&nbsp;`git clone git@github.com:CDCgov/dibbs-cloud.git` <br />  &nbsp;&nbsp;&nbsp;&nbsp;`git clone https://github.com/CDCgov/dibbs-cloud.git`
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.3 Begin Using Terraform
+
+<em><strong>4.3.1.</em></strong> Navigate to terraform/implementation.\
+&ensp; Initialize your local terraform code. `terraform init`\
+<em><strong>4.3.2.</em></strong> Developing in a terraform workspace.\
+&ensp; Check the terraform workspaces. `terraform workspace list`\
+&ensp; _Note_: If you only have a default terraform workspace, you can create a terraform workspace to develop in.  _Skip to "Create a terraform workspace to develop in," below_.
+- <u>Select the terraform workspace to develop in</u>.\
+ &ensp; `terraform workspace select {selectEnvironmentName}`\
+ &ensp; For example, `terraform workspace select dev`.
+- <u>Create a terraform workspace to develop in</u>.\
+&ensp; `terraform workspace new {newEnvironmentName}`.
+&ensp; For example, `terraform workspace new dev`.
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.4 Make A New Branch
+Make a new branch to store any of your amendments to ensure you keep a clean main (or master) branch clear from unapproved revisions.
+
+<em><strong>4.4.1.</em></strong> Navigate to the `dibbs-aws` repository on your local machine.\
+&ensp; `cd /workspace/dibbs-aws`\
+<em><strong>4.4.2.</em></strong> Make a new branch.\
+&ensp; For example, `git checkout -b setup-dibbs-aws-backend-and-services`.
+
+[Return to Table of Contents](#table-of-contents).
+
+## 4.5 Update Terraform Through The Command Line
+
+This section will go over some of the sections you will need to amend or change in your local terraform branch.
+
+<em><strong>4.5.1. Update And Setup Your AWS Backend</em></strong>
+* In your terminal, navigate to the _/terraform/setup_ folder (`cd /terraform/setup`).  
+* Run `./setup.sh`.
+* Now set your variable values for (1) *Region*, (2) *Owner*, and (3) *Project* when prompted in the command line.  
+&nbsp;&nbsp;&nbsp;&nbsp;<em><strong>Note</em></strong>:  The default value for Owner is `Skylight`.  You can change this value to an \
+&nbsp;&nbsp;&nbsp;&nbsp;abbreviated version of your organization (i.e. `WA-STILTS`).  _The Owner name \
```

Comment:
I think the abbreviation you're looking for here is "STLT". This wouldn't necessarily make sense in this context, though...perhaps consider something like "You can change this value to one that represents your organization or department."

Reply:
Great suggestions!  I have updated the documentation to make this clearer for our partners.